### PR TITLE
Remove need for separate `.rubocop_cask.yml`.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

---

`Cask` cops only run on `cask "…" do` blocks, so we can just enable them globally.

### CODE REVIEW

File: Library/.rubocop.yml [hunk 1]
```diff
@@ -2,11 +2,43 @@
 
 require: ./Homebrew/rubocops.rb
 
+inherit_mode:
```
Comment (MikeMcQuaid) [inline Library/.rubocop.yml:5]: TIL!

File: Library/.rubocop.yml [hunk 2]
```diff
@@ -2,11 +2,43 @@
...
   DisplayCopNames: false
   # enable all pending rubocops
   NewCops: enable
+  Exclude:
+    - 'Homebrew/bin/*'
+    - 'Homebrew/vendor/**/*'
+
+Cask/Desc:
+  Description: 'Ensure that the desc stanza conforms to various content and style checks.'
+  Enabled: true
```
Comment (MikeMcQuaid) [inline Library/.rubocop.yml:21]:
```suggestion
Cask:
  Enabled: true
```

Would this be able to replace all those below?

**Review by MikeMcQuaid — APPROVED** [review]
Nice cleanup!

File: Library/.rubocop.yml (hunk 2 above)
Comment (reitermarkus) [reply Library/.rubocop.yml:21]: I'd rather keep them separate for now, so we can maybe use the `config/default.yml` pattern that `rubocop-rspec` uses to enable its default cops.

Comment (MikeMcQuaid) [reply Library/.rubocop.yml:21]: Cool 👍🏻

**Comment by claui** [comment]
This breaks `brew audit Formula/myformula.rb` if you’re in a working copy of a `git clone`d tap.

I don’t speak RuboCop so I tried several shots in the dark, i. e. removing the `Taps/*/*/` part in all the `Exclude` directives. That didn’t work though.

This variant of `brew audit` has worked for years and I’d be super happy to get this working again. @reitermarkus Can you help?

**Comment by reitermarkus** [comment]
@claui what exactly is not working anymore?

Maybe try changing the excludes from `Taps/**/Formula/*.rb` to just `**/Formula/*.rb`.

**Comment by claui** [comment]
> what exactly is not working anymore?

@reitermarkus The audit now reports findings that don’t really make sense for formulae. For example:

```
$ brew audit Formula/node.rb
node:
  * C: 1: col 1: Missing top-level class documentation comment.
  * C: 1: col 1: Missing frozen string literal comment.
  * C: 32: col 3: Assignment Branch Condition size for install is too high. [<3, 33, 2> 33.2/17]
  * C: 32: col 3: Method has too many lines. [14/10]
  * C: 53: col 13: Align the arguments of a method call if they span more than one line.
  * C: 65: col 3: Assignment Branch Condition size for post_install is too high. [<2, 25, 1> 25.1/17]
  * C: 65: col 3: Method has too many lines. [12/10]
  * Formula has other versions so create an alias named node@14.
Error: 8 problems in 1 formula detected
```

**Comment by reitermarkus** [comment]
@claui, remove the `/` in front of the paths.

**Comment by claui** [comment]
@reitermarkus I’ve already tried that but it wouldn’t work because Rubocop [interprets it relative to the YAML file’s location](https://docs.rubocop.org/rubocop/configuration.html#path-relativity).

**Comment by MikeMcQuaid** [comment]
@reitermarkus I think this has made `brew style --fix --display-cop-names homebrew/bundle` now check the `vendor` directory in that tap.

**Comment by reitermarkus** [comment]
@MikeMcQuaid when does it create a `vendor` directory? `bundle install` doesn't create it for me locally.

**Comment by MikeMcQuaid** [comment]
@reitermarkus I've got a global `BUNDLE_PATH: "vendor/ruby"` set which may be related. Ignoring `vendor` would be nice in general, I think, although arguably there should be per-repo configuration there?

**Comment by reitermarkus** [comment]
I actually think I removed it because `vendor/**/*` is ignored by default. Maybe just adding `**/vendor/**/*` will fix it.

**Pull request merged**